meson.build

```diff
@@ -0,0 +1,42 @@
+project('scs', 'c')
+
+py = import('python').find_installation(pure: false)
+
+# This uses the path as is, and avoids running the interpreter.
+incdir_numpy = run_command(py,
+    [
+      '-c',
+      '''import os
+import numpy as np
+incdir = np.get_include()
+print(incdir)
+'''
+    ],
+    check: true
+  ).stdout().strip()
```

Comment:
This is the one annoying thing you have to do indeed - I will get around to replacing all this with a simple `dependency('numpy')` at some point.

For now, I'd recommend to include the `relpath` bit from SciPy: https://github.com/scipy/scipy/blob/166e1f2b1ea0a1a2c3d7b030bd829549f8a5844a/scipy/meson.build#L54-L57

That will ensure things work fine with an in-tree virtual environment (so `numpy` is inside your source tree from Meson's perspective).

Reply:
Ok, I guess fixed (there are more try/except there that I don't understand).